### PR TITLE
synsphenion and intersphenoid suture

### PR DESCRIPTION
**Pull request by megbalk**
issue #2598

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -191,6 +191,7 @@ property_value: doap-SVNRepository https://obo.svn.sourceforge.net/svnroot/obo/u
...
...))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_7500110>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_7500111>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/BFO_0000050>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/RO_0002176>))\n\n############################\n#   Classes\n############################\n\n# Class: <http://purl.obolibrary.org/obo/UBERON_7500110> (<http://purl.obolibrary.org/obo/UBERON_7500110>)\n\nSubClassOf(<http://purl.obolibrary.org/obo/UBERON_7500110> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/UBERON_0006983> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_7500111>)))\n\n# Class: <http://purl.obolibrary.org/obo/UBERON_7500111> (<http://purl.obolibrary.org/obo/UBERON_7500111>)\n\nSubClassOf(<http://purl.obolibrary.org/obo/UBERON_7500111> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/UBERON_0009199> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002176> <http://purl.obolibrary.org/obo/UBERON_0006428>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002176> <http://purl.obolibrary.org/obo/UBERON_0008828>)))\n\n\n)
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:194]: Seeing duplicates in the Anonymous Ancestor:
Screenshot 2022-08-12 at 15 51 43

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -222500,6 +222501,19 @@ intersection_of: connects UBERON:0002396 ! vomer
 relationship: dc-contributor https://orcid.org/0000-0001-8415-4848
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500110
+name: synsphenion
+def: "A craniometric point along the midline of the intersphenoid suture." [ISBN:9780873659505]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:222507]: Is it any point along the midline or a specifically defined point?

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -222500,6 +222501,19 @@ intersection_of: connects UBERON:0002396 ! vomer
...
+relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500111
+name: intersphenoid suture
+def: "A suture joining the basisphenoid bone and the presphenoid bone of a sphenoid bone. This fuses at birth in humans." [ISBN:9780873659505]
+synonym: "intersphenoid synchondrosis" EXACT []
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:222514]: Do you have a dbxref to support this this? Is it the same as the definition?

Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:222514]: "This fuses at birth in humans."

From https://www.ajronline.org/doi/abs/10.2214/ajr.104.1.184 (N.B. from 1968):
"The intersphenoid synchondrosis remains patent in normal children until the age of approximately 3 years."

**Review by bvarner-ebi — COMMENTED** [review]
See inline comments.

File: src/ontology/uberon-edit.obo (hunk 3 above)
Comment (megbalk) [reply src/ontology/uberon-edit.obo:222514]: thank you, I will add the xrefs and will be sure to do that in the future too.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (megbalk) [reply src/ontology/uberon-edit.obo:194]: I think this is the difference in how CARO and UBERON define anatomical entity and anatomical point.

**Comment by megbalk** [comment]
@bvarner-ebi it seems "facial suture" is under "anatomical entity" (CARO) in UBERON. I am not sure why this is, but I can't seem to not have it pop up. "anatomical point" is also an UBERON anatomical entity.

**Comment by megbalk** [comment]
Also, it seems CARO anatomical entity is a subclass of biological entity, then independent continuant. Is there a reason to have them in UBERON?

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (balhoff) [reply src/ontology/uberon-edit.obo:194]: @megbalk no need for the intersection here. Just use two subclass axioms: 
- `'anatomical point'`
- `'part of' some 'inter sphenoid suture'`

**Review by balhoff — CHANGES_REQUESTED** [review]
See inline comment.

**Comment by megbalk** [comment]
fixed the EQ for intersphenoid suture (axiom was originally under "SubClass Of" and not "Equivalent To". Also separated the "SubClass Of" for synsphenion

**Comment by megbalk** [comment]
Note that axioms under "SubClass Of" cannot be nested, e.g., "Class and (property some class) and (property some class)"

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**